main.go

```diff
@@ -115,6 +115,7 @@ func main() {
 	metrics.RegisterCustomMetrics()
 
 	ll.Info("This operator no longer honors the image specified by the custom resources so that it is able to properly coordinate the configuration with the image.")
+	ll.Info("This operator no longer honors the storage specified by the custom resource after it has been created. In order to change storage options, delete the current CR and make a new one.")
```

Comment:
I dont think we need to add this here... the condition should be enough. This message would be shown when the operator starts up and is easily lost. Line 117 actually should have been removed a while ago too :)

Reply:
Got it. I'll go ahead and remove both lines then. :)